show/main.py

```diff
@@ -1240,8 +1240,11 @@ def techsupport(since, verbose, force, yes):
             yes = raw_input('Proceed with any system interruption? (y/n) ')
             if yes.lower() == 'y':
                 cmd += " -f"
-            if yes.lower() == 'n':
+            elif yes.lower() == 'n':
                 exit("Please do \'show techsupport\' without \'--force\'")
+            else:
+                exit("Please try again")
```

Comment:
Why not use `click.confirm()`? https://click.palletsprojects.com/en/7.x/api/?highlight=exit#click.confirm

Reply:
Good information. I didn't know the `click.confirm()`.  I will change to use it.